modules/openapi-generator/src/main/resources/php-laravel/composer.mustache

```diff
@@ -8,59 +8,56 @@
   "license": "MIT",
   "type": "project",
   "require": {
-    "php": "^7.1.3",
-    "fideloper/proxy": "^4.0",
-    "laravel/framework": "5.6.*",
-    "laravel/tinker": "^1.0"
+    "php": "^7.2.5",
+    "fideloper/proxy": "^4.2",
+    "fruitcake/laravel-cors": "^1.0",
+    "guzzlehttp/guzzle": "^6.3",
+    "laravel/framework": "^7.0",
+    "laravel/tinker": "^2.0"
   },
   "require-dev": {
-    "filp/whoops": "^2.0",
-    "fzaninotto/faker": "^1.4",
-    "mockery/mockery": "^1.0",
-    "nunomaduro/collision": "^2.0",
-    "phpunit/phpunit": "^7.0",
-    "phpspec/phpspec": "^4.2",
-    "squizlabs/php_codesniffer": "^3.0"
-  },
-  "autoload": {
-    "classmap": [
-      "database/seeds",
-      "database/migrations",
-      "database/factories"
-    ],
-    "psr-4": {
-      "App\\": "app/",
-      "Tests\\": "tests/"
-    }
-  },
-  "autoload-dev": {
-    "psr-4": {
-      "Tests\\": "tests/"
-    }
-  },
-  "extra": {
-    "laravel": {
-      "dont-discover": [
-      ]
-    }
-  },
-  "scripts": {
-      "post-root-package-install": [
-      "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
-    ],
-      "post-create-project-cmd": [
-      "@php artisan key:generate"
-    ],
-      "post-autoload-dump": [
-      "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
-      "@php artisan package:discover"
-    ]
+    "facade/ignition": "^2.0",
+    "fzaninotto/faker": "^1.9.1",
+    "mockery/mockery": "^1.3.1",
+    "nunomaduro/collision": "^4.1",
+    "phpunit/phpunit": "^8.5"
```

Comment:
```suggestion
    "phpunit/phpunit": "^8 || ^9"
```
It's better to give an option to use latest PHPUnit.

Reply:
The current version of Laravel only recommends `^8.5`, so I wouldn't want to change that.